src/View/Helper/UrlHelper.php

```diff
@@ -57,6 +57,24 @@ public function build($url = null, array $options = []): string
         return $url;
     }
 
+    /**
+     * Returns a URL from a route path string.
+     *
+     * ### Options:
+     *
+     * - `escape`: If false, the URL will be returned unescaped, do only use if it is manually
+     *    escaped afterwards before being displayed.
+     * - `fullBase`: If true, the full base URL will be prepended to the result
+     *
+     * @param string $path Cake-relative route path.
+     * @param array $options Array of options.
+     * @return string Full translated URL with base path.
+     */
```

Comment:
A `@see` tag pointing to `Router::pathUrl()` would be nice so users can easily check what are route paths. Same for the new method in HtmlHelper too.